### PR TITLE
[Docs] iOS updated and removed some old or un-needed docs NR-88745

### PR DESCRIPTION
**Pull request by mbruin-NR**
[NR-88745](https://issues.newrelic.com/browse/NR-88745)
[NR-68044](https://issues.newrelic.com/browse/NR-68044)

After the iOS agent became a .xcframework we didn't need to have separate instructions for tvOS. I removed the duplicate tvOS instructions and made it clear that the iOS instructions can be used.

Added a troubleshooting doc on using Firebase with our agent

Fixed some of the parameters for the sdk api function documentation.  

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by a-sassman** [comment]
@mbruin-NR I've got a relevant PR to change other references of XCFramework/tvOS in the docs: https://github.com/newrelic/docs-website/pull/11528 

When it comes time to release the latest iOS agent, let me know and I'll merge my PR first, then yours.

File: src/nav/mobile-monitoring.yml [hunk 1]
```diff
@@ -295,5 +285,7 @@ pages:
             path: /docs/mobile-monitoring/new-relic-mobile-ios/troubleshoot/event-limits-sampling
           - title: SwiftUI AppDelegate
             path: /docs/mobile-monitoring/new-relic-mobile-ios/troubleshoot/swiftui-appdelegate
+          - title: Firebase Incompatibility
```
Comment (a-sassman) [inline src/nav/mobile-monitoring.yml:288]:
```suggestion
          - title: Firebase incompatibility
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements.mdx [hunk 2]
```diff
@@ -9,7 +9,12 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-tvos-compatibility-requirements
 ---
 
-Before you [install and configure](/docs/mobile-monitoring/new-relic-mobile-ios/tvos/tvos-installation-configuration) the tvOS agent, make sure your tvOS app meets these requirements.
+A tvOS app can install the New Relic agent the same way as iOS. Instructions for installing the agent can be found below.
+ - [Swift Package Manager](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation)
+ - [Cocoa Pods](/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation)
+ - [Manual Installation](/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation)
+
+Before you [install and configure](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation) the tvOS agent, make sure your tvOS app meets these requirements.
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements.mdx:17]:
```suggestion
Before you [install and configure](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation) the iOS agent, make sure your tvOS app meets these requirements.
```

File: src/content/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent.mdx [hunk 3]
```diff
@@ -110,7 +110,7 @@ See the docs for your agent:
 
 * **Android**: [Install](/docs/mobile-apps/android-installation-and-configuration) \| [Release notes](/docs/release-notes/mobile-release-notes/android-release-notes)
 * **iOS**: [Install](/docs/mobile-apps/ios-installation-and-configuration) \| [Release notes](/docs/release-notes/mobile-release-notes/ios-release-notes)
-* **tvOS**: [Update](/docs/mobile-monitoring/new-relic-mobile-ios/tvos/upgrading-new-relic-mobiles-tvos-sdk) \| [Release notes](/docs/release-notes/mobile-release-notes/tvos-release-notes)
+* **tvOS**: [Update](/docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements) \| [Release notes](/docs/release-notes/mobile-release-notes/tvos-release-notes)
```
Comment (a-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent.mdx:113]: * **iOS/tvOS**: [Install](/docs/mobile-apps/ios-installation-and-configuration) \| [Release notes](/docs/release-notes/mobile-release-notes/ios-release-notes)
@mbruin-NR What do you think about this suggestion?

File: src/content/docs/release-notes/mobile-release-notes/ios-release-notes/ios-agent-560.mdx [hunk 4]
```diff
@@ -7,7 +7,7 @@ downloadLink: 'https://download.newrelic.com/ios_agent/NewRelic_iOS_Agent_5.6.0.
 
 ### Improvements
 
-tvOS is now supported! The New Relic agent will now build and run on your tvOS apps with the same great features as the iOS agent! For more information, learn how to [install and configure the tvOS agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/tvos/tvos-installation-configuration).
+tvOS is now supported! The New Relic agent will now build and run on your tvOS apps with the same great features as the iOS agent! For more information, learn how to [install and configure the tvOS agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements).
```
Comment (a-sassman) [inline src/content/docs/release-notes/mobile-release-notes/ios-release-notes/ios-agent-560.mdx:10]:
```suggestion
tvOS is now supported! The New Relic agent will now build and run on your tvOS apps with the same great features as the iOS agent! For more information, learn how to [install and configure the iOS agent for tvOS apps](/docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements).
```

**Comment by a-sassman** [comment]
I moved it out of draft mode to build a preview.

File: src/content/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent.mdx (hunk 3 above)
Comment (mbruin-NR) [reply src/content/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent.mdx]: This change has been made in commit 01bcdbfced193e38a986a5d6aef0e5ad68350ada [01bcdbfced]

**Pull request merged**